packages/design-system/src/styles/components/_Accordion.scss

```diff
@@ -58,7 +58,7 @@
       forced-color-adjust: none;
       -ms-high-contrast-adjust: none;
       outline: $spacer-half solid LinkText;
-      outline-offset: -$spacer-half;
```

Comment:
I think this would have to be converted to `calc(var(--spacer-half) * -1)` once we shift to CSS